components/scream/scripts/jenkins/SCREAM-gpu-ci_setup

```diff
@@ -0,0 +1,7 @@
+# SCREAM-gpu-ci is just a long name for white
+
+export PATH=/ascldap/users/jgfouca/packages/Python-3.6.8-white/bin:$PATH
```

Comment:
Is there a change we can use (also in other machines) the existing modules for python 3, rather than your installation?

Tpls and the likes installed in users directories may get buried deep inside scripts, and become problematic down the road.

I see blake has a python 3.7 module.

Reply:
That's a good idea. I remember trying existing modules and having problems... That could be something we save for a later PR.